README.md

```diff
@@ -1,17 +1,18 @@
 # LocalNetAppChat
 
-LocalNetAppChat (LNAC) is a server/client command line interface that gives your apps a way to communicate with each other localy.
+LocalNetAppChat (LNAC) is a server/client C# command line interface that gives your apps a way to communicate with each other locally.
```

Comment:
Over a network interface I'd say. Maybe over the local network. But locally is like saying they are communicating like usual. Through the cli, which is not the case.